### DISCUSSION TITLE
Signing transactions on Hyperledger Besu

### DISCUSSION THREAD

**Discussion by ghost** [ethereumjs/ethereumjs-monorepo#1563] [Q&A]
Hi team,
Kindly help. How do you sign a transaction on a Hyperledger Besu private network ? This line here:
// The second parameter is not necessary if these values are used
const tx = new EthereumTx(txParams, { chain: 'mainnet', hardfork: 'petersburg' })

**Comment by holgerd77** [comment]
Hi there,
this information you provided here is far from being sufficient, so that it might be possible to give you some help here.

Please give it another thought what information you might be able to provide and what context you can give here and generally give this somewhat more care and time to craft a question.

Thanks! 🙂
